src/asm/rgbasm.1

```diff
@@ -7,89 +7,187 @@
 .\"
 .Dd July 8, 2019
 .Dt RGBASM 1
-.Os RGBDS Manual
+.Os
 .Sh NAME
 .Nm rgbasm
 .Nd Game Boy assembler
 .Sh SYNOPSIS
-.Nm rgbasm
-.Op Fl EhLVvw
+.Nm
 .Op Fl b Ar chars
 .Op Fl D Ar name Ns Op = Ns Ar value
+.Op Fl E
 .Op Fl g Ar chars
+.Op Fl h
 .Op Fl i Ar path
+.Op Fl L
 .Op Fl M Ar dependfile
 .Op Fl o Ar outfile
 .Op Fl p Ar pad_value
 .Op Fl r Ar recursion_depth
-.Ar file
+.Op Fl V
+.Op Fl v
+.Op Fl w
+.Ar
 .Sh DESCRIPTION
 The
 .Nm
-program creates an object file from an assembly source file.
+program creates a RGB object file from an assembly source file.
 The input
 .Ar file
 can be a file path, or
 .Cm \-
 denoting
 .Cm stdin .
-Its arguments are as follows:
+.Pp
+Note that options can be abbreviated as long as the abbreviation is unambiguous:
+.Fl -verb
+is
+.Fl -verbose ,
+but
+.Fl -ver
+could also be
+.Fl -version .
+The arguments are as follows:
 .Bl -tag -width Ds
-.It Fl b Ar chars
+.It Fl b , Fl -binary-digits Ar chars
```

Comment:
`.It Fl b Ar chars , Fl Fl binary-digits Ar chars`

Reply:
Yup, like precedently.

Reply:
Not sure what you mean. I’m saying:

`.It Fl b Ar chars , Fl Fl binary-digits Ar chars`

rather than:

`.It Fl b , Fl -binary-digits Ar chars`

Reply:
Yeah, I meant like [the conversation right above](https://github.com/rednex/rgbds/pull/455#discussion_r355149500).